crates/swc_ecma_transforms_module/src/util.rs

```diff
@@ -46,6 +46,13 @@ pub struct Config {
 
     #[serde(default)]
     pub resolve_fully: bool,
+
+    #[serde(default = "default_js_ext")]
+    pub out_file_extension: String,
+}
+
+pub fn default_js_ext() -> String {
+    "js".to_string()
 }
```

Comment:
```suggestion
impl Config {
    pub fn default_js_ext() -> String {
        "js".to_string()
    }
}
```

Reply:
I've switched the function to an impl on Config and all calls to the function to use the `Config::default_js_ext()` as well.